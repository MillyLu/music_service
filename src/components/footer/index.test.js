/* eslint-disable no-undef */
import { render } from "@testing-library/react";
import { Footer } from "./footer";


describe("<Footer />", () => {
  it("should render successfully and match snapshot", () => {
    const { container } = render(<Footer></Footer>);

    expect(container).toMatchSnapshot();
  });
});