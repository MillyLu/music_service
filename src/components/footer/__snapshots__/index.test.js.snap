// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render successfully and match snapshot 1`] = `
<div>
  <footer
    class="footer"
  />
</div>
`;
